hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -132,7 +132,7 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
   @CommandLine.Option(names = {"--filter"},
       description = "Comma-separated list of \"<field>:<operator>:<value>\" where " +
           "<field> is any valid field of the record, " +
-          "<operator> is (EQUALS,MAX or MIN) and " +
+          "<operator> is (EQUALS,LESSER or GREATER) and " +
```

Comment:
In order to implement MAX and MIN, some code refactoring would be required. We would not be able to do batch processing as max/min cannot be calculated in batches. This could severely affect the performance. So having lesser/greater than operators could perform as a substitute for min/max.

Reply:
Since MIN/MAX operation do not match doing parallel processing of records in batches and generating output, so dropping this. But having alternative with < , >  can be tuned to provide similar functionality of matching.